subsys/net/lib/config/init.c

```diff
@@ -485,9 +485,9 @@ int net_config_init_app(struct device *device, const char *app_info)
 }
 
 #if defined(CONFIG_NET_CONFIG_AUTO_INIT)
-static int init_app(struct device *device)
+static int init_app(struct device *device_config)
```

Comment:
here you use device_config, earlier you used device_entry and other names, maybe just use shorthand dev and be consistent. I do not see how device_config is better here, especially since we have a config member in the device structure.

Reply:
made the change to dev_def